src/components/Movie/Movie.js

```diff
@@ -6,10 +6,10 @@ const Movie = (props) => {
     const { id, image, title } = props
     return(
         <Link to={`/movies/${id}`}>
-        <div className="poster-container">
+        <div className="poster-container" onClick={props.cleanInputs}>
```

Comment:
This will be very helpful to make sure inputs are cleared regardless of where the user clicks.